libs/features/components/banners/src/lib/features-components-banners.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ElewaGroupTeamMemberComponent } from './components/elewa-group-team-member/elewa-group-team-member.component';
+
+@NgModule({
+  imports: [CommonModule],
+  declarations: [ElewaGroupTeamMemberComponent],
+  exports: [ElewaGroupTeamMemberComponent]
+})
+export class FeaturesComponentsBannersModule {}
```

Comment:
Rename this to BannersModule

Reply:
fixed here https://github.com/italanta/elewa-group/pull/132